cci_nepal/utils/dummy_data/dummy_data_utils.py

```diff
@@ -0,0 +1,69 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.13.6
+#   kernelspec:
+#     display_name: cci_nepal
+#     language: python
+#     name: cci_nepal
+# ---
+
+# %load_ext nb_black
+
+
+def label_transformer(d):
```

Comment:
A bit unclear as to why these functions are in utils compared to the tasks in pipeline?

Reply:
Just looking at the utils description in the cookiecutter: 
_This is a place to put utility functions needed across different parts of the codebase.
For example, this could be functions shared across different pieces of analysis or different pipelines._

I guess thats functions that are needed for different tasks? Although it could be argued that you could use this in lots of tests in analysis. I think its just thinking of a consistent approach. So either its alot of functions like this go in utils and pipeline has a script that calls them all and creates the model (whilst also being able to use them ad-hoc in 'analysis' or these all stay in pipeline and utils only holds functions that could be used in different places across the pipeline code (like very generic stuff)